services/github/github-package-registry-downloads.tester.js

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { isMetric, isMetricOverTimePeriod } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+const downloadsToday = Joi.alternatives(
+  isMetricOverTimePeriod,
+  Joi.allow('0/today')
+)
```

Comment:
Apparently `isMetricOverTimePeriod` doesn't support `0` values for the count, but the first few packages I tried to use for these tests had values of `0` for certain intervals. 

Given that GitHub Package Registry is still pretty new/beta, I felt including a `0..` alternative for these tests would help make them less brittle for the short term.